templates/ts/non-sso-tab/vite.config.ts

```diff
@@ -0,0 +1,23 @@
+import { defineConfig } from "vite";
+import { resolve } from "path";
+import commonjs from "vite-plugin-commonjs";
+
+export default defineConfig({
+  plugins: [commonjs()],
```

Comment:
@hund030 If we use the plugin, it will require minimal code changes but will get the warning that Vite's CommonJS (CJS) build is deprecated. If we switch to ESM modules, the project will have both CommonJS and ESM, and converting everything to ESM will result in extensive changes.